openfisca_us/tests/policy/baseline/usda/snap/integration/bplc_ny.yaml

```
@@ -0,0 +1,23 @@
+- name: Family of 1, no member(s) with disabilities, income of 0k/year (Standard Test)
+  period: 2021
+  absolute_error_margin: 0.01
+  input:
+    people:
+      person_1:
+        age: 30
+    spm_units:
+      spm_unit:
+        snap_gross_income: 0 * 12
```

To match the text boxes in the app
```suggestion
        snap_gross_income: 0 * 12  # "Monthly Gross Earned Income"
```

---

done